Makefile

```diff
@@ -58,29 +58,36 @@ config_parser_test: test_setup config_parser.cc config_parser_test.cc
 config_parser_gcov: config_parser_test
 	gcov -r config_parser.cc > config_parser_gcov.txt
 
-http_response_test: test_setup http_response.cc http_response_test.cc
-	g++ $(GCOVFLAGS) $(TESTFLAGS) http_response_test.cc http_response.cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a -o $@ $(LDFLAGS)
+#http_response_test: test_setup http_response.cc http_response_test.cc
```

Comment:
I will keep these in until we fully integrate the new api and remove all the old files